### DISCUSSION TITLE
Finished Booting skill

### DISCUSSION THREAD

**Discussion by C0llab** [jmillerv/mycroft2-skills#3] [General]
I just wanted to say after I mess with the Hello World skill I'm looking at updating the Finished Booting skill here for Dinkum.

**Comment by jmillerv** [comment]
I'll not touch that one. If you do make a PR, in I'll pull it down and test it on mine.

**Comment by jmillerv** [comment]
@C0llab, I'm going to convert that into an issue and when you make the PR, maybe go ahead and just not that your PR closes the issue.
